test/test_events.py

```diff
@@ -23,6 +22,13 @@ def queue_consumer():
         yield mock_ext.return_value
 
 
+@pytest.yield_fixture
+def logger():
+    with patch('nameko.events._log', autospec=True) as patched:
+        yield patched
+    patched.reset_mock()
```

Comment:
isn't the patch is un-applied at this point? though this whole fixture may no longer be used